leshan-core/src/test/java/org/eclipse/leshan/core/node/TimestampedLwM2mNodesTest.java

```diff
@@ -31,13 +32,16 @@
 
 public class TimestampedLwM2mNodesTest {
 
+    private final Instant sampleTimestamp1 = Instant.ofEpochMilli(1_000_000_000);
+    private final Instant sampleTimestamp2 = Instant.ofEpochMilli(2_000_000_000);
```

Comment:
I prefer the previous usage of number instead of variable as you can easily read which one is the most recent timestamp.

But I understand that now this is too verbose.

So maybe we should rename  : 
 - sampleTimestamp1 to ts_1_000_000_000_ms
 - sampleTimestamp2 to ts_2_000_000_000_ms
 
 Or something like this.